src/three-idx.ts

```diff
@@ -232,7 +232,9 @@ export class ThreeIDX {
       if (!this.docs[KEYCHAIN_DEF].metadata.schema) {
         update.metadata = { schema: ThreeIdKeychain }
       }
+      console.log(this.docs[KEYCHAIN_DEF].content)
```

Comment:
logging here, when change fails, these log the same values (can also log update, does exist)